mgmtfn/k8splugin/kubeClient.go

```diff
@@ -103,14 +99,30 @@ func NewAPIClient(serverURL, caFile, keyFile, certFile string) *APIClient {
 	caPool := x509.NewCertPool()
 	caPool.AppendCertsFromPEM(ca)
 
+	// Read client cert
+	cert, err := tls.LoadX509KeyPair(certFile, keyFile)
+	// We need either the client certs or a non-empty authToken to proceed
+	if err != nil {
+		// We cannot use client certs now
+		useClientCerts = false
+		// Check for a non-empty token
+		if len(strings.TrimSpace(authToken)) == 0 {
+			log.Fatalf("%s", err)
```

Comment:
Nit: Can we be a bit more verbose here with the Fatalf. Something like "Neither cert nor token provided".